extensions/common/util.h

```diff
@@ -17,11 +17,28 @@
 
 #include <string>
 
+#include "absl/strings/string_view.h"
+
 namespace Wasm {
 namespace Common {
 
+// None response flag.
+const std::string NONE = "-";
```

Comment:
global variable std::string is not recommended in prod code. The string can be destroyed at exiting, and access log(or any other unknown components) can refers to the heap allocated "-".

Since it's the tech debt of the existing code, could you fix or add a comment to address it later?

Reply:
Done.